scrapy_spider_metadata/_params.py

```diff
@@ -22,11 +22,15 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
     @classmethod
-    def get_param_schema(cls) -> Dict[Any, Any]:
+    def get_param_schema(cls, normalize=False) -> Dict[Any, Any]:
         """Return a :class:`dict` with the :ref:`parameter definition
         <define-params>` as `JSON Schema`_.
 
         .. _JSON Schema: https://json-schema.org/
+
+        If *normalize* is ``True``, the returned schema will be the same
+        regardless of whether you are using Pydantic 1.x or Pydantic 2.x. The
+        normalized schema may not match the output of any Pydantic version.
```

Comment:
Should we state something like "while still implementing the same validation rules" or something like this?

Reply:
https://github.com/scrapy-plugins/scrapy-spider-metadata/pull/4/commits/46dc48c5b199d7b2212e18aaa90ff0ac4a9cdaff ?